Program.cs

```diff
@@ -166,6 +187,13 @@ static void DisplayWord(string wordToGuess, HashSet<char> guessedLetters)
         Console.ResetColor();
     }
 
+    static void DisplayGuessedLetters(HashSet<char> guessedLetters)
```

Comment:
This was a great addition to the code, to visually show the player of this game which letters they couldn't guess more than once.